sharepoint/sharepoint-ps/sharepoint-online/Add-SPOListDesign.md

```diff
@@ -23,7 +24,11 @@ Add-SPOListDesign -Title <String> -SiteScripts <SPOSiteScriptPipeBind[]> [-Descr
 ```
 
 ## DESCRIPTION
-Creates a new list design available to users when they create a new list from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+Creates a new list or document library design available to users when they create a new list or document library from the SharePoint home page, the Microsoft Lists app, Microsoft Teams, or Office.com.
+
+List designs will be available in UI where lists are created, and document library designs will be available in UI where document librraries are created.  The difference is based on the "templateType" of the site design used.  "templateType" 100 is for Lists, "templateType" 101 is for document libraries.
+
+Note that in early 2024, a change will be rolling out which will put document library templates into the document library creation UI and list templates into the list creation UI.  Prior to this change, all designs added through this cmdlet would have appeared in the list creation UI.  Those existing designs will not automatically get moved to document library UI.  To move them to the correct location, you'll need to remove the design and add it again.
 
```

Comment:
it's already 2024, is this line still relevant? If yes, I'd recommend linking out to official documentation page for this feature in learn.microsoft.com